map/map_tests/light_framework_tests.cpp

```diff
@@ -0,0 +1,33 @@
+#include "testing/testing.hpp"
+
+#include "map/light_framework.hpp"
+
+namespace lightweight
+{
+struct LightFrameworkTest
```

Comment:
You can use UNIT_CLASS_TEST here, can't you?

Reply:
I can't declare it as a friend of Framework